google-cloud-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -73,7 +73,7 @@
   private static final String MANIFEST_ARTIFACT_ID_KEY = "artifactId";
   private static final String MANIFEST_VERSION_KEY = "Implementation-Version";
   private static final String ARTIFACT_ID = "google-cloud-core";
-  private static final String LIBRARY_NAME = "gcloud-java";
```

Comment:
Wait..correct me if I am wrong though, `LoggingOptions` is a subclass of `GrpcServiceOptions` which is a subclass of `ServiceOptions`. If I make a change in `ServiceOptions` to set the header, Logging will get the header set automatically.